cli/query/query.go

```diff
@@ -189,3 +199,59 @@ func queryCommittee(ctx *cli.Context) error {
 	}
 	return nil
 }
+
+func queryVoter(ctx *cli.Context) error {
+	args := ctx.Args()
+	if len(args) == 0 {
+		return cli.NewExitError("No address specified", 1)
+	}
+
+	addr, err := flags.ParseAddress(args[0])
```

Comment:
Don't we want to add an address flag instead of an argument?

Reply:
It's mandatory, so why bother with flags? I can't remember all flags, I just want to `query smth for_this_address`.